misc/juliacon2020.md

```diff
@@ -0,0 +1,90 @@
+# JuliaCon 2020 proposal draft
+
+## What to present
+
+My current plan of the presentation consists of two parts:
+
+1. Showcase improvements/features
+- Explain how Juno 1.0 will improve user experiences: easier installation/update and non-interference in user's own environment
+- And also we want to showcase lots of features that only we developers know but would be greatly useful for general users: goto/symbols view/refactor/linter, profiler, etc
+- This part will be of interest of the majority of possible audiences and so should contribute to the whole Julia community
+
+2. Describe implementation
+- In this part, I would like to explain how we solve/implement problems and features; it can help other existing coding environments improve or encourage someone to create another Juno-like tool, or back to our project, attract future contributors and make it easier for them to contribute to Juno
+- Possible presentation flow:
+  * How to separate Juno packages from user's environment: maybe help julia-vscode extension, for example ?
+  * How to implement linter/refactor using both static code analysis and user runtime info: if we can implement "good" features using information that lives in user's runtime, it will show the benefits of Juno's approach in comparison to an ordinal IDE approach based on static code analysis 
+
+Each part will take some time (say, around 10 min), so I think our presentation form is better to be "Talk" (30 mins long including 5 mins for questions) rather than "Lightning Talk" (5 mins long).
+
+## Body
+
+We need to write title/abstract/description to fulfill a submission.
+
+### Submission title
+
+Juno 1.0 – the powerful IDE has got yet more power !
+
+(53/64 chars)
+
+### Submission type
+
+Talk
+
+### Abstract
+
+We will introduce Juno 1.0 -- show how it will get rid of the longstanding issues and bring us huge productivity boosts. 
+Juno's unique approach allows IDE features to be implemented simply but yet with the power of user runtime.
```

Comment:
Maybe we want to make the abstract more concrete on the subject ? I tried not to make it too specific but it now sounds kinda cocky to me ..